adapters/handlers/rest/raft/handler.go

```diff
@@ -154,6 +154,18 @@ func (h RaftHandler) Statistics(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+// RestoreSchemaToV0 migrate from v1 (RAFT) to v0 (Non-RAFT)
+func (h RaftHandler) RestoreSchemaToV0(w http.ResponseWriter, r *http.Request) {
+	restore := h.schemaHandler.RestoreSchemaToV0()
```

Comment:
I suggest using `StoreSchemaV1()` to better reflect that we that a copy of the schema is being written

Reply:
i have been back and forth on that because we have v1 in some places for RAFT, gonna update it and make V1 to V2 for what is raft related